packages/dx-chart-core/src/plugins/layout-manager/computeds.ts

```diff
@@ -18,7 +18,11 @@ export const bBoxes = (prevBBoxes: BBoxes, { bBox, placeholder }: BBoxesChange)
 };
 
 /** @internal */
-export const getRanges = (paneSize: Size): RangesCache => ({
-  [ARGUMENT_DOMAIN]: [0, paneSize.width],
-  [VALUE_DOMAIN]: [paneSize.height, 0],
-});
+export const getRanges = (paneSize: Size, rotated: boolean): RangesCache => {
+  const horRange: NumberArray = [0, paneSize.width];
```

Comment:
It seems that NumberArray should be renamed to NumberTuple

Reply:
Why? Because there are two elements or something else?

Reply:
Yes, `Tuple` implies that it's size is fixed

Reply:
But it does not imply count of 2. It should then be `NumberTuple2` or `Vector2` (like [this](https://docs.microsoft.com/ru-ru/dotnet/api/system.numerics.vector2?view=netframework-4.8)).

However, renaming would add some more files not actually related to rotation to this PR.
Maybe it should be done in a separate PR?